src/assets/EditPenIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconProps } from '@/types/iconProps';
+
+export default function EditPenIcon({ fill, className, onClick, ...props }: IconProps) {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+      onClick={onClick}
+      {...props}
+    >
+      <path
+        d="M13.2451 6.78358C13.2465 6.78216 13.2479 6.78074 13.2494 6.77932C13.259 6.76972 13.2685 6.76014 13.2781 6.75058C13.5685 6.46012 13.8412 6.18734 14.094 5.99232C14.3742 5.77613 14.7228 5.57992 15.17 5.56521C15.2105 5.56388 15.251 5.56388 15.2916 5.56521C15.7387 5.57992 16.0873 5.77613 16.3675 5.99232C16.6204 6.18734 16.893 6.46011 17.1834 6.75056C17.1944 6.76154 17.2054 6.77255 17.2164 6.78357L17.2494 6.81656C17.5399 7.10694 17.8127 7.37964 18.0077 7.63245C18.2239 7.91269 18.4201 8.2613 18.4348 8.70842M13.2451 6.78358L17.5852 8.73636M13.2451 6.78358L6.71375 13.3149M13.2451 6.78358L17.5852 8.8021M18.4348 8.70842L17.5852 8.73636M18.4348 8.70842C18.4348 8.70842 18.4348 8.70841 18.4348 8.7084L17.5852 8.73636M18.4348 8.70842C18.4361 8.74895 18.4361 8.78951 18.4348 8.83004M17.5852 8.73636C17.586 8.75827 17.586 8.78019 17.5852 8.8021M18.4348 8.83004L17.5852 8.8021M18.4348 8.83004C18.4348 8.83005 18.4348 8.83005 18.4348 8.83006L17.5852 8.8021M18.4348 8.83004C18.4201 9.27716 18.2239 9.62577 18.0077 9.90601C17.8127 10.1588 17.5399 10.4315 17.2494 10.7219C17.2384 10.7329 17.2274 10.7439 17.2164 10.7549L10.6851 17.2862C10.6722 17.2991 10.6592 17.3122 10.646 17.3254C10.4923 17.4799 10.3153 17.6577 10.0925 17.7838C9.86977 17.91 9.62626 17.9702 9.41475 18.0226C9.3966 18.0271 9.37868 18.0315 9.36104 18.0359L7.05164 18.6132L7.05163 18.6133L7.01461 18.6225L7.0146 18.6225C7.00375 18.6252 6.99263 18.628 6.98125 18.6309C6.83232 18.6683 6.64031 18.7165 6.47253 18.7329C6.28143 18.7516 5.88674 18.7552 5.56578 18.4342C5.24481 18.1133 5.24843 17.7186 5.26712 17.5275C5.28354 17.3597 5.33174 17.1677 5.36913 17.0187C5.37199 17.0074 5.37478 16.9962 5.37749 16.9854L5.9641 14.639C5.96851 14.6213 5.97294 14.6034 5.97743 14.5853C6.02977 14.3737 6.09003 14.1302 6.21617 13.9074L6.95584 14.3262L6.21617 13.9074C6.34231 13.6847 6.52012 13.5077 6.67456 13.354C6.68781 13.3408 6.70089 13.3278 6.71375 13.3149M6.71375 13.3149L7.31479 13.916L7.19734 13.7985L6.71375 13.3149Z"
+        stroke="#292929"
+        stroke-width="1.7"
+      />
+      <path d="M13 7L17 11" stroke="#292929" stroke-width="1.7" />
```

Comment:
머지하시기 전에 stroke-width만 카멜케이스로 바꿔서 strokeWidth로 바꿔서 추가 커밋 올려주시고 머지해주셔야 콘솔 에러 안 뜰 것 같네요!!
```suggestion
        strokeWidth="1.7"
      />
      <path d="M13 7L17 11" stroke="#292929" strokeWidth="1.7" />
```